specs/posix/docker_container_processes.table

```diff
@@ -2,8 +2,8 @@ table_name("docker_container_processes")
 description("Docker container processes.")
 schema([
     Column("id", TEXT, "Container ID", index=True, required=True),
-    Column("pid", BIGINT, "Process ID"),
-    Column("name", TEXT, "The process path or shorthand argv[0]", index=True),
+    Column("pid", BIGINT, "Process ID", index=True),
```

Comment:
I don't see this used as an index. Maybe additional?

Reply:
The combination of `id` and `pid` are the `PRIMARY KEY` for this table.